src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/FunctionItemCallIterator.java

```diff
@@ -37,11 +37,10 @@
 import java.util.List;
 import java.util.Map;
 
-public class UserDefinedFunctionCallIterator extends HybridRuntimeIterator {
+public class FunctionItemCallIterator extends HybridRuntimeIterator {
 
     private static final long serialVersionUID = 1L;
     // parametrized fields
-    private FunctionIdentifier _functionIdentifier;
```

Comment:
Can you move _functionitem(defined below) to thisline to keep consistency with the comments regarding parametrized fields or calculated fields. If these comments are not helpful, you can also remove them if you wish

Reply:
Done! Good catch. The comments make sense.